### PR TITLE
Add um_forcing and vm_forcing to the appropriate tendencies

### PR DESCRIPTION
**Pull request by bstephens82**
...for CLUBB+MF.  These are a total kludge and should not be considered part of the standard CLUBB source code.  One file is modified for the case where clubb_l_diag_Lscale_from_tau=.false. and another file is for the case when this flag is .true.

### CODE REVIEW

File: advance_windm_edsclrm_module.F90 [hunk 1]
```diff
@@ -1395,7 +1395,7 @@ subroutine compute_uv_tndcy( solve_type, fcor, perp_wind_m, perp_wind_g, xm_forc
 
       end select
 
-      xm_tndcy(1:gr%nz) = xm_gf(1:gr%nz) + xm_cf(1:gr%nz)  &
+      xm_tndcy(1:gr%nz) = & !xm_gf(1:gr%nz) + xm_cf(1:gr%nz)  &
```
Comment (adamrher) [inline advance_windm_edsclrm_module.F90]: would it be less invasive to not change the l_implemented logic an instead just modify the else case:

      xm_tndcy = 0.0_core_rknd

setting it instead to ```xm_tndcy(1:gr%nz) = xm_forcing(1:gr%nz)``` ?

File: advance_xm_wpxp_module.F90 [hunk 2]
```diff
@@ -2527,12 +2527,12 @@ subroutine solve_xm_wpxp_with_single_lhs( dt, l_iter, nrhs, wm_zt, wp2, tau_C6_z
        ! and ( iiPDF_type == iiPDF_ADG1 .or. iiPDF_type == iiPDF_new_hybrid )
 
        ! Coriolis term for <u> and <v>
-       if ( .not. l_implemented ) then
+       if ( l_implemented ) then
```
Comment (adamrher) [inline advance_xm_wpxp_module.F90]: similar here, the else case has:

          um_tndcy = zero
          vm_tndcy = zero

Doing it here would avoid changing the l_implemented logic, no?

**Review by adamrher — CHANGES_REQUESTED** [review]
Hi Ben, seeing that you've changed the l_implemented logic, it looks like you could do these same mods w/o changing the l_implemented logic, and just filling in the else case with the mods. Maybe I'm missing something, but if not, I think Id rather not change the logic.

**Comment by bstephens82** [comment]
Sure I can set it up that way.  I think at first I thought I might need to make some kind of "case" logic in advance_windm_edsclrm, but after looking more closely I think that doesn't matter since we're not dealing with Coriolis or geostrophic forcings.

File: advance_windm_edsclrm_module.F90 (hunk 1 above)
Comment (bstephens82) [reply advance_windm_edsclrm_module.F90]: Done.

File: advance_xm_wpxp_module.F90 (hunk 2 above)
Comment (bstephens82) [reply advance_xm_wpxp_module.F90]: Done.

Comment (adamrher) [reply advance_xm_wpxp_module.F90]: for this one can you be explicit with the indexing ```(1:gr%nz)```, like it does in the l_implemented block?

Comment (bstephens82) [reply advance_xm_wpxp_module.F90]: Done.

Comment (bstephens82) [reply advance_xm_wpxp_module.F90]: Well, actually I'm not seeing it.  Hmm.

Comment (bstephens82) [reply advance_xm_wpxp_module.F90]: I'm confused, becuase I pushed the change to my branch, and if you click on my branch it's there, but for some reason is not showing up in the PR.  Any ideas?

Comment (adamrher) [reply advance_xm_wpxp_module.F90]: I had to hit that blue button by your last commit entry that says "see new changes" and then it applied.

Comment (bstephens82) [reply advance_xm_wpxp_module.F90]: OK it showed up.  Done.

**Pull request merged**